requirements.txt

```diff
@@ -1,34 +1,23 @@
-bump2version==0.5.11
-wheel==0.33.6
-watchdog==0.9.0
-flake8==3.8.4
-mypy==0.790
-tox==3.14.0
-coverage==4.5.4
-Sphinx==1.8.5
-twine==1.14.0
-Click==7.1
+bump2version
+wheel
+watchdog
+flake8
+mypy
+tox
+coverage
+twine
```

Comment:
May be able to remove twine, it's used for package deployment but I'm not sure if we use it. If it doesn't exist in any of the Make targets, you can probably remove it.

Reply:
I was thinking that. Removed.